src/rospkg/os_detect.py

```diff
@@ -73,9 +73,11 @@ def read_issue(filename="/etc/issue"):
 
 def read_os_release(filename="/etc/os-release"):
     """
-    :returns: Dictonary of key value pairs from /etc/os-release, with quotes stripped from values
+    :returns: Dictonary of key value pairs from /etc/os-release or fallback to /usr/lib/os-release, with quotes stripped from values
```

Comment:
Can you not just pass `filename='/usr/lib/os-release'` to the `FdoDetector` for Clear Linux?

That's why we have the parameter for non-standard locations.

Reply:
Sorry after reading more in https://github.com/clearlinux/distribution/issues/180#issuecomment-419133986 , the fallback behavior is expected of os-release